experimental/sdg-tracker/src/components/layout/AppHeader.tsx

```diff
@@ -15,116 +15,39 @@
  */
 
 import { Link } from "react-router-dom";
-import styled from "styled-components";
 import { BrandingLink } from "./components";
+import "./AppHeader.css";
+import React from "react";
 
 type HeaderOptions = "home" | "countries" | "goals" | "topics" | "search";
 
-const HeaderContainer = styled.div`
-  width: 100%;
-  background: white;
-`;
+interface Props {
+  children: React.ReactNode;
+  to?: any;
+  selected?: boolean;
+}
 
-const TopContainer = styled.div`r
-  align-items: center;
-  border-bottom: 1px solid #f1f1f1;
-  display: flex;
-  flex-direction: row;
-  height: 75px;
-  justify-content: space-between;
-  padding: 0 4rem;
-  width: 100%;
-`;
+const newNode = (className: string) => {
```

Comment:
Instead of creating the new node here, i think it would be more clear to just replace these custom classes `<TopContainer />`, etc with divs directly in the AppHeader definition

E.g.,

```
const AppHeader = (props: { selected: HeaderOptions }) => {
  const { selected } = props;
  return (
    <div className="header-container">
      <div className="top-container">
      ...
```

Reply:
Done.